.changeset/lazy-ducks-pay.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
Oops, we need this one and the one below back

Reply:
You need the previous changesets?

Reply:
Yes, we need to keep existing changesets un-touched and then just generate one changeset for this change only